modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -170,7 +170,7 @@ You can define more than one secure configuration properties file to read proper
 
 == File Level Encryption
 
-Since version 1.1.0 of the Secure Configuration Properties module, you can encrypt the entire file (not only the values of the properties). To use a file that its encrypted, you add the `fileLevelEncryption` attribute and set it to `true`: 
+Since version 1.1.0 of the Secure Configuration Properties module, you can encrypt the entire file (not only the values of the properties). To use a file that its encrypted, you add the `fileLevelEncryption` attribute and set it to `true`:
```

Comment:
Not a biggie, but you don't need "you add.." just "add" works fine,

Reply:
this is old content. The only change was the spacing after the ending colon.